client/components/Photos.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```

Comment:
Is it necessary to have an extra file that only places an image?

Reply:
I prefer doing it this way mostly because if I have time, I want to refactor so that the component can display multiple photos. Having this extra file will make that refactor easier and currently doesn't make things any harder.